deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -496,3 +496,13 @@ def __on_pre_torrent_removed(self, torrent_id):
                         'Failed to removed torrent file "%s" from "%s": %s',
                         torrent_fname, copy_torrent_path, ex,
                     )
+
+    @export
+    def is_admin_level(self):
+        rpcserver = component.get('RPCServer')
```

Comment:
assign getting the component in the plugin __init__ to `self.rpcserver` and it can be reused in the entire class.

Reply:
Done.